src/workerd/api/pyodide/setup-emscripten.c++

```diff
@@ -72,6 +72,7 @@ EmscriptenRuntime EmscriptenRuntime::initialize(
   kj::Maybe<capnp::Data::Reader> emsciptenSetupJsReader;
   kj::Maybe<capnp::Data::Reader> pythonStdlibZipReader;
   kj::Maybe<capnp::Data::Reader> pyodideAsmWasmReader;
+  js.installJspi();
```

Comment:
IMHO this fits better inside workerd-api and internalrepo-api instead, it's kind hard to find when it's here.
Though this is mostly a nit

Reply:
nvm, i get it now, ignore me